src/pkg/clouds/aws/ecs/cfn/setup.go

```diff
@@ -228,6 +229,13 @@ func (a *AwsEcs) TearDown(ctx context.Context) error {
 		return err
 	}
 
+	// Disable termination protection before deleting the stack
```

Comment:
Oops, I forgot about this commit. It's to make it harder to manually delete the CD stack from the AWS console.